lldb/source/Plugins/Process/AIX/NativeProcessAIX.h

```diff
@@ -0,0 +1,119 @@
+//===-- NativeProcessAIX.h ---------------------------------- -*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LIBLLDB_NATIVEPROCESSAIX_H_
+#define LIBLLDB_NATIVEPROCESSAIX_H_
```

Comment:
This is the header guard for elf core. I'd try to match that:
`LLDB_SOURCE_PLUGINS_PROCESS_ELF_CORE_PROCESSELFCORE_H`

(the linux file still uses the old header probably guard because Jonas used some tool which only updates the files that are actually used in the build (and the linux file is not built on a mac)